client/src/modules/patients/record/patient-record.html

```diff
@@ -105,13 +105,21 @@
                   <span translate>FORM.LABELS.TAKE_A_PICTURE</span>
                 </a>
 
-                <a
+                <a ng-if="!PatientRecordCtrl.patient.employee_uuid"
                   data-method="convert-to-employee"
                   ui-sref="patientAsEmployee({ uuid : PatientRecordCtrl.patient.uuid })"
                   class="btn btn-default btn-block">
                   <span class="fa fa-street-view"></span>
                   <span translate>FORM.LABELS.REGISTER_EMPLOYEE_PATIENT</span>
                 </a>
+
+                <span ng-if="PatientRecordCtrl.patient.employee_uuid && !PatientRecordCtrl.patient.locked" class="btn btn-success btn-block" translate>
```

Comment:
I would prefer if this didn't look like a button.  I tried to click it and was confused why nothing happened.  Can you just make it a `<p class="text-success">`?